Owncloud iOs Client/Owncloud iOs Client-Info.plist

```diff
@@ -98,6 +102,7 @@
 	<array>
 		<string>audio</string>
 		<string>fetch</string>
+		<string>location</string>
```

Comment:
Please, remove this location key and try it without it, should work without it and Apple does not like this.